### PR TITLE
Fix install destination

### PR DESCRIPTION
**Pull request by YutoUchimi**
Install ...
- programs in `scripts/` and `spacenav/` into ${CATKIN_PACKAGE_BIN_DESTINATNION}
- `config`, `launch`, `models`, `resource`, `sample`, `sample_scripts` and `test` dir into ${CATKIN_PACKAGE_SHARE_DESTINATION}.

cf.
http://wiki.ros.org/catkin/CMakeLists.txt
http://www.ros.org/reps/rep-0122.html

### CODE REVIEW

**Comment by YutoUchimi** [comment]
@knorth55 Could you please review this if you have time?

File: jsk_rviz_plugins/CMakeLists.txt [hunk 1]
```diff
@@ -185,16 +185,25 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
...
+install(FILES plugin_description.xml
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
+install(DIRECTORY cfg config icons launch samples test
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
-  USE_SOURCE_PERMISSIONS)
+  USE_SOURCE_PERMISSIONS
+)
+
 install(DIRECTORY ./src/
```
Comment (knorth55) [inline jsk_rviz_plugins/CMakeLists.txt]: `.src/` -> `src`

**Comment by knorth55** [comment]
same as https://github.com/jsk-ros-pkg/jsk_recognition/pull/2345#issuecomment-432142374

File: jsk_rviz_plugins/CMakeLists.txt (hunk 1 above)
Comment (YutoUchimi) [reply jsk_rviz_plugins/CMakeLists.txt]: Thanks! Fixed now.

**Comment by YutoUchimi** [comment]
Follow this guideline: https://github.com/jsk-ros-pkg/jsk_recognition/pull/2345#issuecomment-434154926

**Pull request merged**